content/docs/reference/cookies.mdx

```diff
@@ -161,6 +161,12 @@ cookie:
 
 ## Cookie Secure {#cookie-secure}
 
+:::caution
+
+As of v0.25.0, we have deprecated the **Cookie Secure** setting in Pomerium Core, the Enterprise Console, and the Ingress Controller and will remove it in **v0.26.0**. It will be enabled by default going forward.
```

Comment:
nit: the phrase "by default" doesn't seem quite accurate if you can't change it at all; maybe something like "Pomerium will always set the `Secure` attribute on session cookies going forward."